src/renderer/bootstrap.tsx

```diff
@@ -27,10 +27,22 @@ export async function bootstrap(App: AppComponent) {
     themeStore.init(),
   ]);
 
+  // Register additional store listeners
+  clusterStore.registerIpcListener();
+
   // init app's dependencies if any
   if (App.init) {
     await App.init();
   }
+  window.addEventListener("message", (ev: MessageEvent) => {
+    if (ev.data === "teardown") {
```

Comment:
This event currently is bound to both apps (ClusterManager & Dashboard), but should be listening only in iframes.
Probably make sense move to `app.tsx` or add extra check `process.isMainFrame` before binding.

Also maybe we should check for `evt.origin` that it comes from parent host (`localhost:port`)
Read more: https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage